enhancements/sha-bundle-digest.md

```diff
@@ -0,0 +1,136 @@
+---
+title: SHA-Bundle-Digest
+authors:
+  - '@theishshah'
+reviewers:
+  - '@camilamacedo86'
+  - '@joelanford'
+  - '@jmrodri'
+  - '@ryantking'
+  - '@tlwu2013'
+approvers:
+  - '@tlwu2013'
+  - '@jmrodri'
+  - '@joelanford'
+
+creation-date: 2022-01-14
+
+last-updated: 2022-01-14
+
+status: implementable
+---
+
+# Support SHA-based Image Tags
+
+## Summary
+
+Currently, operators generated by `operator-sdk` will store and retrieve container images based on the image name and
+the image tag in the bundle. This EP adds support to `operator-sdk` such that images can be stored and retrieved in the
+bundle with an image name and an image digest as an optional replacement for a tag. Image digests are SHA-256 sums that
+are hashed from the image layer therefore uniquely identifying one specific image.
+
+## Motivation
+
+Many users run Kubernetes in an air-gapped environment (also known as an offline or disconnected environment), which is
+when a cluster network does not have any access to the public internet, usually for security purposes. In this setup,
+the digest of an image will be different than it is in the public repository since the image location changes. For
+example, if a user imports the image `quay.io/golang:1.17` into their air-gapped registry as
+`quay.internaldomain.net/golang:1.17`, the digest of the image will be different due to the different name even though
+the contents of the images are identical. Using digest SHAs instead of tags in the air-gapped environment will keep the
+digest from changing registry to registry.
+
+### Goals
+
+- Give users the ability to enable digest-based image resolution
+- Allow users to specify additional images to resolve digests for.
+- Add ability to resolve image digests from image tag dynamically using local docker registry.
```

Comment:
This feels like a proposal. Replacing with different goals that I think are missing.
```suggestion
- Use image digests in the output `ClusterServiceVersion` (see [Appendix B](#appendix-b)).
    - Images specified in deployments will have digests.
    - The `spec.relatedImage` section of the CSV will contain a mapping of images to their digests.
```